posthog/session_recordings/test/__snapshots__/test_session_recordings.ambr

```diff
@@ -912,111 +912,62 @@
 # ---
 # name: TestSessionRecordings.test_get_session_recordings.27
   '''
-  SELECT "posthog_grouptypemapping"."id",
```

Comment:
this seems highly sus

Reply:
Ah no - it's good actually, the massive diff is caused by fewer postgres queries, which is a win!